build.bat

```diff
@@ -21,7 +21,7 @@ rem Build with nuget, it solves the directory structure for us.
 call .\Tools\nuget\build.bat %ARCH_OPT% %PGO_OPT% %REBUILD_OPT%
 
 rem Install with nuget into a build folder
-.\externals\windows-installer\nuget\nuget.exe install python -Source %~dp0\PCbuild\amd64 -OutputDirectory nuget-result
+if "%ARCH_OPT%" EQU "-x64" (.\externals\windows-installer\nuget\nuget.exe install python -Source %~dp0\PCbuild\amd64 -OutputDirectory nuget-result) else (.\externals\windows-installer\nuget\nuget.exe install python -Source %~dp0\PCbuild\win32 -OutputDirectory nuget-result)
```

Comment:
Also, it would be best if you could split this out to separate lines for cleanliness.